lib/linguist/languages.yml

```
@@ -4193,6 +4193,15 @@ Module Management System:
   tm_scope: none
   ace_mode: text
   language_id: 235
+Mojo:
+  type: programming
+  color: "#ff4c1f"
+  extensions:
+  - ".mojo"
+  - ".\U0001F525"
+  ace_mode: text
```

```suggestion
  ace_mode: python
  codemirror_mode: python
  codemirror_mime_type: text/x-python
```

Mojo is similar enough to Python that updates are pulled from [`MagicPython.tmBundle`](https://github.com/MagicStack/MagicPython/blob/master/grammars/MagicPython.tmLanguage), according to [comments](https://github.com/modularml/mojo-syntax/blob/81cf73ef2d4a1b149d58b4e7fab2f1acf4edde2d/syntaxes/mojo.syntax.json#L2-L6) in its grammar file. Ergo, we might as well repurpose the Python modes of CodeMirror and ACE (as the alternative here would be no syntax highlighting in those editors at all).

---

Done, thanks for your help! (We've also refreshed the samples included to use the latest Mojo syntax 😄)

---

CodeMirror's [`python` mode](https://codemirror.net/5/mode/python/index.html) does a decent enough job of highlighting Mojo source, but it hasn't been included in the PR as I initially suggested.

~~~diff
 Mojo:
   type: programming
   color: "#ff4c1f"
   extensions:
   - ".mojo"
   ace_mode: python
   tm_scope: source.mojo
   language_id: 1045019587
+  codemirror_mode: python
+  codemirror_mime_type: text/x-python
~~~

@modocache or @lattner, could one of you please action this remaining change?

---

Yes, can do right now, thanks! Sorry, missed this comment when I commented on the main thread earlier. Thanks!

---

Done -- sorry again for missing this!

---

All good! Now we're ready to rock. 😉